src/stories/ActionMenu.stories.tsx

```diff
@@ -0,0 +1,174 @@
+              description: 'Create a brand new Codespace with a fresh image and checkout this branch.',
+              descriptionVariant: 'block'
+            }
+          ]}
+        />
+      </ErsatzOverlay>
+    </>
+  )
+}
+ActionsStory.storyName = 'Actions'
+
+export function SimpleListStory(): JSX.Element {
+  return (
+    <>
+      <h1>Simple List</h1>
+      <ErsatzOverlay>
+        <ActionMenu
+          buttonContent="Menu"
+          items={[
+            {text: 'New file'},
+            ActionList.Divider,
+            {text: 'Copy link'},
+            {text: 'Edit file'},
+            {text: 'Delete file', variant: 'danger'}
+          ]}
+        />
+      </ErsatzOverlay>
+    </>
+  )
+}
+SimpleListStory.storyName = 'Simple List'
+
+export function ComplexListStory(): JSX.Element {
+  return (
+    <>
+      <h1>Complex List</h1>
+      <ErsatzOverlay>
+        <ActionMenu
+          buttonContent="Menu"
+          groupMetadata={[
+            {groupId: '0'},
+            {groupId: '1', header: {title: 'Live query', variant: 'subtle'}},
+            {groupId: '2', header: {title: 'Layout', variant: 'subtle'}},
+            {groupId: '3', renderItem: props => <ActionList.Item style={{fontWeight: 'bold'}} {...props} />},
+            {groupId: '4'}
+          ]}
+          items={[
+            {leadingVisual: TypographyIcon, text: 'Rename', groupId: '0'},
+            {leadingVisual: VersionsIcon, text: 'Duplicate', groupId: '0'},
+            {
+              leadingVisual: SearchIcon,
+              text: 'repo:github/memex,github/github',
+              groupId: '1',
+              renderItem: props => <ActionList.Item style={{color: 'rebeccapurple'}} {...props} />,
+              onClick: () => console.log('Item 1')
+            },
+            {
+              leadingVisual: NoteIcon,
+              text: 'Table',
+              description: 'Information-dense table optimized for operations across teams',
+              descriptionVariant: 'block',
+              groupId: '2',
+              onClick: () => console.log('Item 2')
+            },
+            {
+              leadingVisual: ProjectIcon,
+              text: 'Board',
+              description: 'Kanban-style board focused on visual states',
+              descriptionVariant: 'block',
+              groupId: '2',
+              onClick: () => console.log('Item 3')
+            },
+            {
+              leadingVisual: FilterIcon,
+              text: 'Save sort and filters to current view',
+              groupId: '3'
+            },
+            {leadingVisual: FilterIcon, text: 'Save sort and filters to new view', groupId: '3'},
+            {leadingVisual: GearIcon, text: 'View settings', groupId: '4'}
+          ]}
+        />
+      </ErsatzOverlay>
+    </>
+  )
+}
+ComplexListStory.storyName = 'Complex List'
+
+export function CustomTrigger(): JSX.Element {
+  const customAnchor = (props: any) => <Link {...props} />
```

Comment:
Making a note on this - ideally we should just be able to pass `Link` directly to `renderAnchor`, but that doesn't seem to work in the way I thought it would.